### ISSUE TITLE
Training values fluctuating a lot

### ISSUE THREAD

**Issue by MasIgor** [ultralytics/yolov5#7097]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello! 
I am trying to train 19 classes. the training is VERY fast. in about 10/15 epochs it reaches 0.99. 
Then after some epochs it starts jumping up and down like in the screenshot:

![image](https://user-images.githubusercontent.com/67956327/159421319-d5f001a6-6abc-4d50-aad2-8be87358d5f3.png)

What could be the cause of it? i am using Scratch-low as hyperparameters, starting with no weights. 
Also, in order to reduce the learning rate, should the value be higher or lower in the hyp file? what does "high-medium-low" even refer to?

Thank you!

**Comment by glenn-jocher** [comment]
@tanzerlana low statistics datasets can expect high noise, and the end of the warmup period may produce temporary instabilities.

**Comment by MasIgor** [comment]
thank you. my dataset is low (450 images/class) because all images of all classes are taken in the very same position, and the object fills up about 80% of the image. all with the same orientation and all with the same light. for this reason I thought that a relatively small dataset should be enough. I tried also with 3k images per class, but it did not make any real difference in the result.
